### DISCUSSION TITLE
Trouble getting Update Inventory to work

### DISCUSSION THREAD

**Discussion by Twinkbug** [etsy/open-api#973] [Q&A]
Despite many different attempts I cannot get a successful response when using the updateInevntory call

I get the  response  "error : invalid Json string."  No other info is given
 I've simplified it to just a simple listing with  a quantity change

The following is set to the Product  parameter when making the call 
 {"sku":"4309E-CRM","offerings":[{"Price":8.45,"quantity":20,"is_enabled":1}]}]

I am using RestSharp for the PUT call.
Other Get and Post calls I have used work fine - no authentication issues.

Any help or suggestions to try would be appreciated!

**Comment by JAY6390** [comment]
No doubt you can thin it down, but this is how I got it to work

```{"products":[{"offerings":[{"quantity":286,"is_enabled":true,"price":7.99}],"property_values":[]}],"price_on_property":[],"quantity_on_property":[],"sku_on_property":[],"listing":null}```

**Comment by nk-gears** [comment]
> Despite many different attempts I cannot get a successful response when using the updateInevntory call
> 
> I get the response "error : invalid Json string." No other info is given I've simplified it to just a simple listing with a quantity change
> 
> The following is set to the Product parameter when making the call {"sku":"4309E-CRM","offerings":[{"Price":8.45,"quantity":20,"is_enabled":1}]}]
> 
> I am using RestSharp for the PUT call. Other Get and Post calls I have used work fine - no authentication issues.
> 
> Any help or suggestions to try would be appreciated!

Were you able to resolve this. I am also getting same error. No matter whatever i passed.
